psi4/src/read_options.cc

```diff
@@ -1439,6 +1439,11 @@ int read_options(const std::string &name, Options &options, bool suppress_printi
         /*- When using |scf__stability_analysis| ``FOLLOW``, maximum number of orbital optimization attempts
             to make the wavefunction stable. !expert -*/
         options.add_int("MAX_ATTEMPTS", 1);
+        /*- Do Perform Incremental Fock Build? -*/
+        options.add_bool("INCFOCK", false);
+        /*- Frequency with which to compute the full Fock matrix if using |scf__incfock| . 
+        N means rebuild every N SCF iterations to avoid accumulating error from the incremental procedure. -*/
+        options.add_int("INCFOCK_FULL_FOCK_EVERY", 5);
```

Comment:
Just as a heads up, I was able to run the IncFock code with values of INCFOCK_FULL_FOCK_EVERY below 1. Running negative values for this option seems to perform incfock for all iterations, while setting this option to 0 leads to a floating-point exception on my system. It may be wise to add some sort of option checking to make sure that the input value for this variable makes sense.

Reply:
Just fixed